auth/auth0/README.md

```diff
@@ -6,6 +6,7 @@ Add Auth0 Authentication to your Graphcool project 🎁
 ## Getting Started
 
 ### 1. Add the template to your Graphcool service
+From your server directory (e.g. if you were to just have run `graphcool init server` your "server directory" would be `./server`) run:
```

Comment:
I would say 'service directory', as the standard terminology is 'Graphcool service'.